src/cbmc/bmc.cpp

```diff
@@ -223,6 +224,14 @@ void bmct::report_success()
       std::cout << "\n";
     }
     break;
+
+  case ui_message_handlert::JSON_UI:
+    {
+      json_objectt json_result;
+      json_result["CProverStatus"] = json_stringt("Success");
```

Comment:
I'm ok either way, but I think consistency is key. That is, at least within one function or, better, one file, the same rules should be followed. I'd suggest adding a whitespace-changes-only commit doing that.